lambdas/replay-cimit-vcs/dynamo-scripts/scan.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+Help()
+{
+   # Display Help
+   echo "Syntax: scriptTemplate [-e|p]"
+   echo "options:"
+   echo "e     Specifies the environment label for the script output"
+   echo "p     Specifies the AWS profile to use with the script"
+   echo
+}
+
+# Set variables
+Env="build"
+Profile="None"
+
+# Script options
+while getopts ":he:p:" option; do
+   case $option in
+      h) # display Help
+         Help
+         exit;;
+      e) # Enter an environment
+         Env=$OPTARG;;
+      p) # Enter an AWS profile
+         Profile=$OPTARG;;
+     \?) # Invalid option
+         echo "Error: Invalid option"
+         exit;;
+   esac
+done
+
+echo Performing scan for $Profile in $Env
```

Comment:
Call me paranoid, do you think it's worth showing a warning before using this script? Something like 'this may take several minutes and use up read capacity etc., do you wish to continue? (Y/n)'

Alternatively, putting a similar warning in the README?

Reply:
Good idea - it's pretty quick on build, but that obviously won't be the case in prod